src_server/Api.re

```diff
@@ -0,0 +1,34 @@
+open Express;
+
+let dirname =
+  switch ([%bs.node __dirname]) {
+  | Some(d) => d
+  | None => Js.Exn.raiseError("Could not get __dirname")
+  };
+
+let getRefdomainsJson = () => {
+  Node.Fs.readFileAsUtf8Sync(Node.Path.join([|dirname, "Refdomains.json"|]))
```

Comment:
Probably, not a big deal, but I recommend all-lowercase filename to avoid possible issues with case-insensitive filesystems.